UKHomeOffice/sgmr-service
# Cg 121 check page and edit

TO TEST

- go to /reports
- if you have none, create a report
- click on the link on the date
> you should see the check details page
> this page should show the saved details of your voyage (for newer voyages this will only be the departure details

----

**I can edit a voyage**

- click 'Change' next to departure details
> you should be taken to the departure details page with your voyage details prepopulated
- change something (add/remove/edit)
- click save and continue
> you should see a blank page 2
- go back to /reports
- click on your voyage
> you should see the check details page with your changes

----

NOTES
- page 2-5 not yet available
- arrivalDate will show Invalid Date on check page as it is not yet handled in this voyage
